carma_wm_ctrl/src/WMBroadcasterNode.cpp

```diff
@@ -156,30 +156,34 @@ carma_ros2_utils::CallbackReturn WMBroadcasterNode::handle_on_activate(const rcl
   timer_ = create_timer(get_clock(),
                           std::chrono::milliseconds((int)(config_.traffic_control_request_period * 1000)),
                           std::bind(&WMBroadcasterNode::spin_callback, this));
-  
+
   /////////////
   //SUBSCRIBERS
   ///////////// NOTE: subscriber declaration delayed until here so that when map is received, publisher is already activated to immediately publish back
 
   // Base Map Sub
   base_map_sub_ = create_subscription<autoware_lanelet2_msgs::msg::MapBin>("base_map", 1, std::bind(&WMBroadcaster::baseMapCallback, wmb_.get(), std_ph::_1));
-  
+
   // Base Map Georeference Sub
   georef_sub_ = create_subscription<std_msgs::msg::String>("georeference", 1, std::bind(&WMBroadcaster::geoReferenceCallback, wmb_.get(), std_ph::_1));
-  
+
   // Geofence Sub
-  geofence_sub_ = create_subscription<carma_v2x_msgs::msg::TrafficControlMessage>("geofence", 200, std::bind(&WMBroadcaster::geofenceCallback, wmb_.get(), std_ph::_1));
-  
+  rclcpp::SubscriptionOptions geofence_sub_options;
```

Comment:
Can we add a comment here explaining why we have to disable intra-process communications for this subscriber that is now transient local? We can copy the comment from here: https://github.com/usdot-fhwa-stol/carma-platform/blob/d385568f9f10c695f547c9bf58be68cf89ea0058/carma_wm/src/WMListener.cpp#L134C1-L135C1